UPGRADING

```diff
@@ -767,6 +767,9 @@ PHP 8.4 UPGRADE NOTES
 - MySQLnd:
   . Improved the performance of MySQLnd quoting.
 
+- PCRE:
+  . Improved the performance of named capture groups.
```

Comment:
in the 1st commit - https://github.com/php/php-src/pull/14424/commits/569c29172929fdd33037fe87cb080c3d15bae8b3 - wrong GH issue is mentioned

Reply:
Multitasking gone wrong, I'll fix it on merge when I squash this.